### PR TITLE
Megatron test patch

### PR DESCRIPTION
**Pull request by ericharper**
Patching megatron test.

### CODE REVIEW

File: nemo/collections/nlp/modules/common/megatron/megatron_utils.py [hunk 1]
```diff
@@ -136,12 +137,14 @@ def get_megatron_checkpoint(pretrained_model_name):
         path (str): path to model checkpoint
     '''
     url = MEGATRON_CONFIG_MAP[pretrained_model_name]['checkpoint']
-    path = os.path.join(MEGATRON_CACHE, pretrained_model_name)
+    cache_dir = os.path.join(MEGATRON_CACHE, pretrained_model_name)
+    path = os.path.join(cache_dir, 'model_optim_rng.pt')
```
Comment (ekmb) [inline nemo/collections/nlp/modules/common/megatron/megatron_utils.py:141]: filename defined already contains .pt. For non model-parallel checkpoint, we're asking users to provide path to the file not dir

Comment (ericharper) [reply nemo/collections/nlp/modules/common/megatron/megatron_utils.py:141]: that wasn't the case for me, path was just the directory. pretrained_model_name is just "megatron-bert-345m-uncased"

Comment (ekmb) [reply nemo/collections/nlp/modules/common/megatron/megatron_utils.py:141]: "megatron-bert-345m-uncased" is supposed to be a filename, if you use 'model_optim_rng.pt' cased and uncased models will be confused

Comment (ekmb) [reply nemo/collections/nlp/modules/common/megatron/megatron_utils.py:141]: does torch/megatron dir exist? @ericharper

Comment (ericharper) [reply nemo/collections/nlp/modules/common/megatron/megatron_utils.py:141]: oh okay, should we put a .pt or something on it? To make that more clear?

Comment (ericharper) [reply nemo/collections/nlp/modules/common/megatron/megatron_utils.py]: MEGATRON_CACHE doesn't necessarily exist.

**Pull request merged**